### PR TITLE
Added debugging steps for Visual Studio

### PR DESCRIPTION
**Pull request by Reezaali**
Added debugging steps in case the personality menu is not available.

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/attach-debugger-from-task-pane.md](https://github.com/OfficeDev/office-js-docs-pr/blob/Reezaali-patch-1/docs/testing/attach-debugger-from-task-pane.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810171934061579-481%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=jgVy%2Bj2z7ykiyKoF35NpL1oRsTXlFMPkphbFU1v5wGY%3D&st=2018-10-17T19%3A30%3A24Z&se=2018-11-17T19%3A35%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/attach-debugger-from-task-pane.md [hunk 1]
```diff
@@ -22,6 +22,11 @@ Select **Attach Debugger**. This launches the **Visual Studio Just-in-Time Debug
 
 In Visual Studio, you will see the code files in **Solution Explorer**.   You can set breakpoints to the line of code you want to debug in Visual Studio.
 
+If you don't see the Personality menu, you can debug your add-in using Visual Studio. Ensure your task pane add-in is open in Office, and then follow these steps:
+
+1. In Visual Studio, choose **DEBUG** > **Attach to Process**.
+2. In **Attach to Process**, choose all of the available Iexplore.exe processes, and then choose the **Attach** button.
+
```
Comment (AlexJerabek) [inline docs/testing/attach-debugger-from-task-pane.md]: Should this be in a note section, since it's a unexpected case?

Comment (Reezaali) [reply docs/testing/attach-debugger-from-task-pane.md]: @AlexJerabek - added the note.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/attach-debugger-from-task-pane.md](https://github.com/OfficeDev/office-js-docs-pr/blob/Reezaali-patch-1/docs/testing/attach-debugger-from-task-pane.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810171945501852-481%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=WvrAcxLR3RqNkVB0lOfxbfEodpoj3oYA8QJbdYML7n8%3D&st=2018-10-17T19%3A42%3A07Z&se=2018-11-17T19%3A47%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**